src/packet/header/header_parser.rs

```diff
@@ -1,6 +1,6 @@
 pub trait HeaderParser {
     type Output;
 
-    /// Write the header to the given buffer.
-    fn parse(&self, mut buffer: &mut Vec<u8>) -> Self::Output;
+    // Write the header to the given buffer.
```

Comment:
Could you explain why you removed the doccomment?

Reply:
strange, can we disable that clippy warning?

here is an example of docstrings on a trait https://github.com/tower-rs/tower/blob/master/tower-service/src/lib.rs#L201